src/traces/scatter3d/index.js

```diff
@@ -14,7 +14,11 @@ Scatter3D.plot = require('./convert');
 Scatter3D.attributes = require('./attributes');
 Scatter3D.markerSymbols = require('../../constants/gl3d_markers');
 Scatter3D.supplyDefaults = require('./defaults');
-Scatter3D.colorbar = require('../scatter/marker_colorbar');
+Scatter3D.colorbar = {
+    container: 'marker | line',
```

Comment:
Hmm. That ` | ` business is a little too hacky for my taste. 

Could you make this:

```js
 Scatter3D.colorbar = [{
  container: 'marker',
  min: 'cmin',
  max: 'cmax'
}, {
  container: 'line',
  min: 'cmin',
  max: 'cmax'
}]
```

? Thank you!